src/EnergyPlus/MixedAir.cc

```diff
@@ -3204,6 +3204,9 @@ namespace MixedAir {
 		// without AirloopHVAC objects).
 		if ( InitOAControllerSetUpAirLoopHVACVariables ) {
 			if ( NumPrimaryAirSys > 0 ) {
+				if ( !allocated( AirLoopZoneInfo ) ){
+					AirLoopZoneInfo.allocate( NumPrimaryAirSys );
```

Comment:
This allocate lets it run without crashing but there won't be any data in AirLoopZoneInfo for the code below if get here before it's been allocated.